InitialDatabaseSQL.sql

```diff
@@ -0,0 +1,245 @@
+
+use Test;
+
+DROP TABLE address_info;
```

Comment:
For organization, have all the drop tables at the top as such,

drop table hotel_chain;
drop table hotel_phone;

drop table renting_info;
drop table booking_info;
drop table customer;
drop table role;

drop table room;
drop table hotel;
drop table employee;
drop table room_category;
drop table address_info;